library/lua/gui.lua

```diff
@@ -923,17 +923,26 @@ local function make_frame(name, double_line)
     return frame
 end
 
-WINDOW_FRAME = make_frame('Window', true)
-PANEL_FRAME = make_frame('Panel', false)
-MEDIUM_FRAME = make_frame('Medium', false)
-BOLD_FRAME = make_frame('Bold', true)
-INTERIOR_FRAME = make_frame('Thin', false)
-INTERIOR_FRAME.signature_pen = false
-INTERIOR_MEDIUM_FRAME = copyall(MEDIUM_FRAME)
-INTERIOR_MEDIUM_FRAME.signature_pen = false
+FRAME_WINDOW = make_frame('Window', true)
+FRAME_PANEL = make_frame('Panel', false)
+FRAME_MEDIUM = make_frame('Medium', false)
+FRAME_BOLD = make_frame('Bold', true)
+FRAME_INTERIOR = make_frame('Thin', false)
+FRAME_INTERIOR.signature_pen = false
+FRAME_INTERIOR_MEDIUM = copyall(FRAME_MEDIUM)
+FRAME_INTERIOR_MEDIUM.signature_pen = false
 
 -- for compatibility with pre-steam code
-GREY_LINE_FRAME = WINDOW_FRAME
+GREY_LINE_FRAME = FRAME_PANEL
```

Comment:
I'd keep this as pointing to FRAME_WINDOW since that was its original purpose -- as the external frame for an entire tool window.

Reply:
FRAME_PANEL is also for external window, but it's the one without the thick title bar and little draggable corner. It's the one meant for panels that can't be resized/moved.

Like, FRAME_WINDOW is for draggable/resizable panels.

Reply:
the "original" windows weren't draggable or resizable, but they still need the thick title bar, since they're for top-level windows, right?

Reply:
not necessarily. :thinking: Originally (in .47) they all just used the thick character for all sides... right?

Like, the thick bar is thick because it's supposed to be grabbed. If the window can't be moved, it shouldn't be there!
It's like the hotkeys menu frame, that one can't be moved and lacks a thick/title bar

Reply:
ok, that's fair. it will still have the "title" area in either case, which is fine. ok. approved. thanks!